src/App.tsx

```diff
@@ -5,7 +5,11 @@ import { FindMovie } from './components/FindMovie';
 import { Movie } from './types/Movie';
 
 export const App = () => {
-  const [movies] = useState<Movie[]>([]);
+  const [movies, setMovies] = useState<Movie[]>([]);
+
+  const addMovieToTheList = (movie: Movie) => {
+    setMovies([...movies, movie]);
```

Comment:
Consider using a callback inside setMovies and disabling adding existing movies
<img width="977" alt="Screenshot 2023-08-30 at 16 45 48" src="https://github.com/mate-academy/react_movies-list-fetch-movies/assets/135232001/8e9bd7a6-8ce3-4729-a045-874154f34d05">